queues.md

```diff
@@ -591,6 +591,8 @@ Alternatively, you may run the `queue:listen` command. When using the `queue:lis
 
     php artisan queue:listen
 
+Another benefit of `queue:listen` is that it features a higher degree of job isolation, which prevents changes made by a job from being propagated to subsequent ones in certain scenarios, e.g. static variables mutations.
+
```

Comment:
Personally I don't like this "pro argument" for `queue:listen`.

IMHO `queue:work` is absolutely to be preferred (e.g. if you "go horizon", it's not possible to do otherwise AFAIK).

Rather I think the possible pitfall for `queue:work` should be pointed out clearer.

:-)

Reply:
@mfn Just pushed an update. How about it now?

Reply:
> non-idempotent mutations

**I** ❤️ it, thanks!